prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -25,6 +25,7 @@ import gov.cdc.prime.router.Schema
 import gov.cdc.prime.router.SettingsProvider
 import gov.cdc.prime.router.Source
 import gov.cdc.prime.router.ValueSet
+import gov.cdc.prime.router.azure.WorkflowEngine
```

Comment:
This feels like a violation of the separation of concerns. You shouldn't need to query the DB within a serializer. That is the work to give the serializer all the data it needs to make should be done beforehand.